Failed: MultiDcHeartbeatTakingOverSpec

https://jenkins.akka.io:8498/job/akka-multi-node-repeat/14117/consoleFull

```
[JVM-5] - must be healthy (on node 'fifth', class akka.cluster.MultiDcHeartbeatTakingOverSpecMultiJvmNode5) *** FAILED *** (13 milliseconds)
[JVM-5]   java.lang.AssertionError: assertion failed: expected class akka.cluster.CrossDcHeartbeatSender$MonitoringActive, found class akka.cluster.CrossDcHeartbeatSender$MonitoringDormant (MonitoringDormant())
[JVM-5]   at scala.Predef$.assert(Predef.scala:170)
[JVM-5]   at akka.testkit.TestKitBase$class.expectMsgClass_internal(TestKit.scala:490)
[JVM-5]   at akka.testkit.TestKitBase$class.expectMsgType(TestKit.scala:471)
[JVM-5]   at akka.testkit.TestKit.expectMsgType(TestKit.scala:831)
[JVM-5]   at akka.cluster.MultiDcHeartbeatTakingOverSpec$$anonfun$1$$anonfun$apply$mcV$sp$2$$anonfun$apply$mcV$sp$4.apply$mcV$sp(MultiDcHeartbeatTakingOverSpec.scala:120)
```

--- a/akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcHeartbeatTakingOverSpec.scala
+++ b/akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcHeartbeatTakingOverSpec.scala
@@ -109,19 +109,25 @@ abstract class MultiDcHeartbeatTakingOverSpec extends MultiNodeSpec(MultiDcHeart
       enterBarrier("found-expectations")
     }
 
-    "be healthy" taggedAs LongRunningTest in {
+    "be healthy" taggedAs LongRunningTest in within(5.seconds) {
       implicit val sender = observer.ref
       runOn(expectedAlphaHeartbeaterRoles.toList: _*) {
-        selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatus()
-        observer.expectMsgType[CrossDcHeartbeatSender.MonitoringActive](5.seconds)
+        awaitAssert {
+          selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatus()
+          observer.expectMsgType[CrossDcHeartbeatSender.MonitoringActive]
+        }
       }
       runOn(expectedBetaHeartbeaterRoles.toList: _*) {
-        selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatus()
-        observer.expectMsgType[CrossDcHeartbeatSender.MonitoringActive](5.seconds)
+        awaitAssert {
+          selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatus()
+          observer.expectMsgType[CrossDcHeartbeatSender.MonitoringActive]
+        }
       }
       runOn(expectedNoActiveHeartbeatSenderRoles.toList: _*) {
-        selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatus()
-        observer.expectMsgType[CrossDcHeartbeatSender.MonitoringDormant](5.seconds)
+        awaitAssert {
+          selectCrossDcHeartbeatSender ! CrossDcHeartbeatSender.ReportStatus()
+          observer.expectMsgType[CrossDcHeartbeatSender.MonitoringDormant]
+        }
       }
 
       enterBarrier("sunny-weather-done")

